modules/ovis/src/ovis.cpp

```diff
@@ -453,7 +453,7 @@ class WindowSceneImpl : public WindowScene
         if(tus->getTextureName() != name)
         {
             RTShader::ShaderGenerator::getSingleton().invalidateMaterial(
-                RTShader::ShaderGenerator::DEFAULT_SCHEME_NAME, *bgplane->getMaterial());
+                RTShader::ShaderGenerator::DEFAULT_SCHEME_NAME, bgplane->getMaterial()->getName());
```

Comment:
you should also pass `RESOURCEGROUP_NAME`, so the Material is uniquely identified

Reply:
Done